intermediate_source/forward_ad_usage.py

```diff
@@ -172,5 +172,75 @@ def jvp(ctx, gI):
                          check_backward_ad=False, check_undefined_grad=False,
                          check_batched_grad=False)
 
+######################################################################
+# Functional API
+# --------------------------------------------------------------------
+# We also offer a higher-level functional API in functorch
+# for computing Jacobian-vector products that you may find simpler to use
+# depending on your use case.
+#
+# The benefit of the functional API is that there isn't a need to understand
+# or use the lower-level dual tensor API and that you can compose it with
+# other functorch transforms (like vmap); the downside is that it offers you
+# less control.
+#
+# Note that the remainder of this tutorial will require functorch
+# (https://github.com/pytorch/functorch) to run. Please find installation
+# instructions at the specified link.
+
+import functorch as ft
+
+primal0 = torch.randn(10, 10)
+tangent0 = torch.randn(10, 10)
+primal1 = torch.randn(10, 10)
+tangent1 = torch.randn(10, 10)
+
+def fn(x, y):
+    return x ** 2 + y ** 2
+
+# Here is a basic example to compute the JVP of the above function.
+# The jvp(func, primals, tangents) returns func(*primals) as well as the
+# computed jvp. Each primal must be associated with a tangent of the same shape.
+primal_out, tangent_out = ft.jvp(fn, (primal0, primal1), (tangent0, tangent1))
+
+# functorch.jvp requires every primal to be associated with a tangent.
+# If we only want to associate certain inputs to `fn` with tangents,
+# then we'll need to create a new function that captures inputs without tangents:
+primal = torch.randn(10, 10)
+tangent = torch.randn(10, 10)
+y = torch.randn(10, 10)
+
+import functools
+new_fn = functools.partial(fn, y=y)
+primal_out, tangent_out = ft.jvp(new_fn, (primal,), (tangent,))
+
+######################################################################
+# Using the functional API with Modules
+# --------------------------------------------------------------------
+# To use ``nn.Module`` with functorch.jvp to compute jvps with respect to the
```

Comment:
nit: JVPs (or writing it out as Jacobian-vector product?) just so that its different than lower case `jvp`, the API